zipkin-receiver-kafka/README.md

```diff
@@ -20,6 +20,25 @@ $ KAFKA_ZOOKEEPER=127.0.0.1:2181 bin/collector
 $ KAFKA_ZOOKEEPER=127.0.0.1:2181 KAFKA_TOPIC=notzipkin bin/collector
 ```
 
+### Encoding spans into Kafka messages
+
+The message's binary data includes a list header followed by N spans serialized in TBinaryProtocol
+
+```
+write_byte(12) // type of the list elements: 12 == struct
+write_i32(count) // count of spans that will follow
+for (int i = 0; i < count; i++) {
+  writeTBinaryProtocol(spans(i))
+}
+```
+
+If using [zipkin-java](https://github.com/openzipkin/zipkin-java), `Codec.THRIFT.writeSpans(spans)`
+implements the above.
+
+#### Legacy encoding
+Versions before 1.35 accepted a single span per message, as opposed to a list per message. This
+practice is deprecated, but still supported.
```

Comment:
ps there isn't a likely scenario when we will remove support for single-span-per-message prior without a new model (like zipkin 2). In other words, this is just a request to please not write new code that does this.